glide.yaml

```diff
@@ -0,0 +1,21 @@
+package: github.com/intelsdi-x/snap-plugin-processor-movingaverage
+import:
+- package: github.com/intelsdi-x/snap
+  version: dbbb1ae3ef7282ac02bd6e14ef44d93fb7c9e85b
+  subpackages:
+  - control/plugin
+  - core
+  - pkg/ctree
+  - pkg/schedule
+  - scheduler/wmap
+  - control/plugin/cpolicy
+  - core/ctypes
+- package: github.com/robfig/cron
```

Comment:
This also seems to be dependency of some other package (snap?). Remove if that's the case